index.html

```diff
@@ -1,13 +1,30 @@
 <!doctype html>
 <html lang="en">
-  <head>
+<head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite App</title>
-  </head>
-  <body>
-    <div id="app"></div>
-    <script type="module" src="/main.js"></script>
-  </body>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+<div id="app">
+    <button id="button1" class="tooltip-container">
+        Button 1
+        <div id="tooltip1" class="tooltip">Hello World</div>
+    </button>
+    <button id="button2" class="tooltip-container">
+        Button 2
+        <div id="tooltip2" class="tooltip">Hello World</div>
```

Comment:
If you have structure where you have a button and tooltip inside, you can find tooltip just by class name, without adding an id.
`document.querySelector('#button1 .tooltip')`